docs/concepts/00_what_is_aries.md

```diff
@@ -0,0 +1,36 @@
+# What is Aries?
+
+Hyperledger Aries provides a shared, reusable, interoperable tool kit designed for initiatives and solutions focused on creating, transmitting and storing verifiable digital credentials. It is infrastructure for blockchain-rooted, peer-to-peer interactions. This project consumes the cryptographic support provided by Hyperledger Ursa, to provide secure secret management and decentralized key management functionality. (1)
+
+## Where did Aries come from?
+Hyperledger Aries is related to both Hyperledger Indy, which provides a resolver implementation, and Hyperledger Ursa, which it uses for cryptographic functionality. Aries will consume the cryptographic support provided by Ursa to provide both secure secret management and hardware security modules support. (2)
+
+## Key Characteristics
+
+- A blockchain interface layer (known as a resolver) for creating and signing blockchain transactions
```

Comment:
A resolver can be seen as part of the larger thing known as the VDRI:

> Aries Verifiable Data Registry Interface: An interface for verifying data against an underlying ledger.

Reply:
i'll add this too